From 1ed669c95b5ed72fe82e725c1d3be8fc16de5717 Mon Sep 17 00:00:00 2001
From: Etienne Cordonnier <ecordonnier@snap.com>
Date: Wed, 19 Apr 2023 18:02:21 +0200
Subject: [PATCH] adb: Allow adbd to be run as root

Signed-off-by: Etienne Cordonnier <ecordonnier@snap.com>
---
 packages/modules/adb/daemon/main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/modules/adb/daemon/main.cpp b/packages/modules/adb/daemon/main.cpp
index 0e01bde6..7ddfefe9 100644
--- a/packages/modules/adb/daemon/main.cpp
+++ b/packages/modules/adb/daemon/main.cpp
@@ -64,6 +64,7 @@
 static const char* root_seclabel = nullptr;
 
 static bool should_drop_privileges() {
+    return true;
     // The properties that affect `adb root` and `adb unroot` are ro.secure and
     // ro.debuggable. In this context the names don't make the expected behavior
     // particularly obvious.
-- 
2.36.1.vfs.0.0

